src/hooks/useErrorNote/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export interface ErrorNoteData {
+  title: string;
+  description: string;
+}
+
+interface ErrorNoteContextProps {
+  error: ErrorNoteData | null;
+  setErrorNote: (message: ErrorNoteData) => void;
+  clearErrorNote: () => void;
+}
+
+const ErrorNoteContext = React.createContext<ErrorNoteContextProps | undefined>(undefined);
+
+export const ErrorNoteProvider = ({ children }: { children: React.ReactNode }) => {
```

Comment:
In my current task where I'm trying that AuthContextProvider stuff, I created a new directory `src/providers` and put it there. I still left the hook part in the `useAuth`.

Maybe that "path" could be used here also?